queries.js

```diff
@@ -0,0 +1,34 @@
+const { Pool, Client } = require('pg')
+const connectionString = 'postgresql://localhost:5432/reviewsdb?user=ethan&password=ethan';
+
+const pool = new Pool({
+  connectionString: connectionString,
+})
+
+const getReviews = (res) => {
+  // add WHERE to retrieve reviews per product_id
+  // when dealing with pagination and count, add LIMIT
+  // when testing performance, take limit off to see what happens when asking for all from the set
```

Comment:
I really like there comments

Reply:
no, I haven't dealt with those yet, but I'll be happy to have these notes when I do :)